src/components/Profile/styled.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+const StyledProfile = styled.section`
+	margin: 0 auto;
+
+	#userInformation {
+		display: flex;
+		flex-direction: column;
+		justify-content: center;
+	}
+	button#bigEditButton {
+		font-size: 1rem;
+	}
```

Comment:
I would recommend to not use `id`s for styling. Usually it is considered kind of bad practice and in the long run it won't be scale-able since you can only give an `id` once.

Reply:
This is in an component this isn´t meant to be scalable, due to its a one of a kind element.
But do you have another suggestion on how to style these?
Build an extra component for each button?

Reply:
You could use a class here instead. Might be the nicer approach in this case than an id.

In your button component you already have styles for the `editBigButton` defined:

```
			${props =>
		props.editButtonBig
			? css`
					font-size: 1rem;
					color: var(--dark);
			  `
			: null}
```
So you could just add a `editButtonBig` prop to your button and the styles should apply.